### DISCUSSION TITLE
crossorigin="anonymous"

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1086] [Anything Else]
With the changes to the script last week in order to introduce a proxy, the tracking has stopped for the sites that used the not-documented crossorigin="anonymous" in their snippet. Couple of people have been in touch about it and simple removing crossorigin="anonymous" fixes it. We didn't officially support crossorigin="anonymous" but just adding here as information if someone else stumbles on it.

**Comment by michael-voit** [comment]
Is crossorigin="" working? Or do you not support it at all? Webhint see's that as an error, so I look for a solution.

**Reply by metmarkosaric** [reply]
crossorigin="anonymous" is not supported
